docs/sql-ref-ansi-compliance.md

```diff
@@ -380,6 +380,10 @@ When ANSI mode is on, it throws exceptions for invalid operations. You can use t
   - `try_element_at`: identical to the function `element_at`, except that it returns `NULL` result instead of throwing an exception on array's index out of bound.
   - `try_to_timestamp`: identical to the function `to_timestamp`, except that it returns `NULL` result instead of throwing an exception on string parsing error.
   - `try_parse_url`: identical to the function `parse_url`, except that it returns `NULL` result instead of throwing an exception on url parsing error.
+  - `try_make_timestamp`: identical to the function `make_timestamp`, except that it returns `NULL` result instead of throwing an exception on string parsing error.
+  - `try_make_timestamp_ltz`: identical to the function `make_timestamp_ltz`, except that it returns `NULL` result instead of throwing an exception on string parsing error.
+  - `try_make_timestamp_ntz`: identical to the function `make_timestamp_ntz`, except that it returns `NULL` result instead of throwing an exception on string parsing error.
```

Comment:
> ... an exception on string parsing

Sorry, I didn't get which strings does the function parse? It builds a timestamps from datetime fields, no?

Reply:
removed the string parsing part